packages/layout/src/tree-layout/drawing-manager/drawingManager.js

```diff
@@ -0,0 +1,127 @@
+import { HTMLRenderer } from '../renderers/html-renderer';
+import { Utils } from '../utils/utils';
+
+export class DrawingManager {
+    constructor (data, renderer, container) {
+        this.data = data.tree;
+        this.componentMap = data.componentMap;
+        this.layoutClassName = data.layoutClassName;
+        this.renderer = renderer;
+        global.__renderer = renderer; // TODO change global into diff place
+        this.mount = Utils.isDOMElement(container) ? container : Utils.getElement(container);
+    }
+
+    _drawLayout () {
```

Comment:
Reason for _ in method name? 

If the method has to be private, write a function and call the function by passing the required arguments.

Reply:
function is private to the class .. no use of declaring outside.